src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx

```diff
@@ -37,7 +36,7 @@ const toBlockInfo = (current, value) => ({ ...current, [value.id]: blockStateFac
 
 const paginationParams = {
   page: 1,
-  page_size: 20,
+  page_size: +process.env.LIBRARY_AUTHORING_PAGINATION_PAGE_SIZE,
```

Comment:
```suggestion
  page_size: process.env.LIBRARY_AUTHORING_PAGINATION_PAGE_SIZE,
```
Should there be this `+`?

Reply:
Symbol `+` parse to number, I think that for future work it is better to use explicit cast to number type